stl/inc/xutility

```diff
@@ -3487,7 +3487,7 @@ namespace ranges {
 
     // clang-format off
     template <size_t _Idx, class _It, class _Se, subrange_kind _Ki>
-        requires (_Idx < 2)
+        requires ((_Idx  == 0 && copyable<_It>) || _Idx == 1)

```

Comment:
There's an extra space here. (The peril of disabling clang-format! :scream_cat:)